bib/cmd/bootc-image-builder/main.go

```diff
@@ -554,6 +536,10 @@ func run() error {
 	manifestCmd.Flags().StringArray("type", []string{"qcow2"}, fmt.Sprintf("image types to build [%s]", allImageTypesString()))
 	manifestCmd.Flags().Bool("local", false, "use a local container rather than a container from a registry")
 	manifestCmd.Flags().String("rootfs", "", "Root filesystem type. If not given, the default configured in the source container image is used.")
+	err := manifestCmd.Flags().MarkDeprecated("local", "please ensure the container image has already been pulled")
```

Comment:
This emits a warning when the user passes it, right?  I feel like it's going to be annoying for people...let's just silently accept it for a while; like at least on the order of ~months until we have a chance to update the docs?

Reply:
Sure, I'll update in a bit. Just wanted to see if the tests all pass first in CI